### PR TITLE
Fiexed some errors with the package

### PR DESCRIPTION
**Pull request by stplasim**
- Fixed import errors
- Fixed typescript compilation error
- Fixed linting errors

### CODE REVIEW

File: generator.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ function formatName(string) {
 function componentTemplate(name, svg) {
   let component = "import React from 'react'\n"
   component += "import { IconProps } from './types'\n"
-  component += `const ${name}: React.FC<IconProps> = (props) => (`
+  component += `const ${name}: React.FC<IconProps> = (props: IconProps):any => (`
```
Comment (alexlanz) [inline generator.js]: Are you sure that this is required? @mmalfertheiner do we need here a return value?

File: src/components/Icon360.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { IconProps } from './types'
+const Icon360: React.FC<IconProps> = (props: IconProps): any => (
```
Comment (alexlanz) [inline src/components/Icon360.tsx]: Could we add an additonal empty line between the imports and the component declaration?

File: package.json [hunk 3]
```diff
@@ -50,7 +53,9 @@
     "eslint-plugin-prettier": "^3.1.4",
     "prettier": "^2.1.2",
     "ts-jest": "^26.4.1",
-    "typescript": "^4.0.3"
+    "typescript": "^4.0.3",
+    "react": "^16.13.1",
```
Comment (alexlanz) [inline package.json]: Why is react and react-com here included twice?

**Review by alexlanz — CHANGES_REQUESTED** [review]

File: generator.js (hunk 1 above)
Comment (mmalfertheiner) [reply generator.js]: Simon, React.FC is an interface, which defines the parameters and the return value. Thats why the original version was correct.

**Pull request merged**